auth/http.go

```diff
@@ -598,10 +602,12 @@ func getUserObjectsJSON(central *ImqsCentral, users []authaus.AuthUser, ident2pe
 		} else if err != nil {
 			central.Central.Log.Warnf("issue fetching group names for user %v : %v", user.UserId, err)
 		}
-		userStats, err := central.Central.GetUserStats(user.UserId)
+
 		if err != nil {
 			return nil, err
 		}
```

Comment:
The `err` was relevant to the old `GetUserStats`. If we leave it in, it will conflict with the warning logging on 603 and actually return now.